Helper/Ratings.php

```diff
@@ -126,7 +126,7 @@ public function getRatings(Product $product, Store $store)
      */
     private function getRatingsFromProviders(Product $product, Store $store)
     {
-        if ($this->nostoDataHelper->isRatingTaggingEnabled($store)) {
+        if ($this->nostoDataHelper->isRatingTaggingEnabled($store) && $this->isReviewModuleEnabled()) {
```

Comment:
We cannot do this. Magento's review module is disabled and for example Yotpo is enabled we would return `null`